src/components/Footer/Footer.jsx

```diff
@@ -14,8 +14,9 @@ export default function Footer() {
           </p>
           <ContactForm />
         </div>
-        <ul>
-          <li>GoodEWorkers © 2023</li>
+        <ul className="footerEnd">
+          <li className="copy">GoodEWorkers © 2023</li>
+          <li><a href="/legals">Legal Disclaimers</a></li>
```

Comment:
For link between pages we prefer using : https://www.gatsbyjs.com/docs/linking-between-pages/ for optimization